gpd_ros2/CloudIndexed cloud_indexed
---
gpd_ros2/GraspConfigList grasp_configs
